src/runtime/internal/Component.ts

```diff
@@ -22,9 +22,11 @@ interface T$$ {
 }
 
 export function bind(component, name, callback) {
-	if (component.$$.props.indexOf(name) === -1) return;
-	component.$$.bound[name] = callback;
-	callback(component.$$.ctx[name]);
+	if (component.$$.props.hasOwnProperty(name)) {
```

Comment:
Linting doesn't like using `Object.prototype.hasOwnProperty` apparently - and probably for good reason, as I think this would get messed up with a prop called `hasOwnProperty`. What I has meant was actually using `Object.hasOwnProperty(component.$$.props, name)`

Reply:
wait, when did `hasOwnProperty` become a static method on `Object`? It works in Chrome, but does it work everywhere? I always used to do `Object.prototype.hasOwnProperty.call(obj, prop)`

Reply:
Apparently I'm going crazy. I just found [this SO question](https://stackoverflow.com/q/58554462), asked a mere three days ago. It sounds like `Object.hasOwnProperty` doesn't really exist, but, because of javascript mysteries, you can call it, but it doesn't actually do what I was expected. Sorry about that! `Object.prototype.hasOwnProperty.call(component.$$.props, name)` sounds safest.

Reply:
Turns out we're using `hasOwnProperty` somewhere already so I turned it into a `has_prop` helper